### PR TITLE
ci: label issue with version

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

As we introduced label option #17226 we want also parse this information to label so we can filter issues

<details>
  <summary><b>Before submitting</b></summary>

- Was this **discussed/agreed** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- Did you make sure to **update the documentation** with your changes? (if necessary)
- Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- Did you list all the **breaking changes** introduced by this pull request?
- Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

</details>

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

<details>
  <summary>Reviewer checklist</summary>

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

</details>

cc @borda @carmocca

### CODE REVIEW

**Review by awaelchli — APPROVED** [review]

File: .github/labeler.yml [hunk 1]
```diff
@@ -31,3 +31,6 @@ ci:
   - '.github/**'
   - 'dockers/**'
   - '.pre-commit-config.yml'
+
+docs:
```
Comment (carmocca) [inline .github/labeler.yml:35]: We generally only use the docs label for PRs that only focus on docs
But many PRs do a small docs modification. This change would add it to them.

Do you want this?

File: .github/workflows/issue-labeler.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        uses: stefanbuck/github-issue-parser@v3
+        id: issue-parser
+        with:
+          template-path: .github/ISSUE_TEMPLATE/1_bug_report.yaml
+
+      - name: Set labels based on severity field
+        uses: redhat-plumbers-in-action/advanced-issue-labeler@v2
+        with:
+          issue-form: ${{ steps.issue-parser.outputs.jsonString }}
+          section: versions
```
Comment (carmocca) [inline .github/workflows/issue-labeler.yml:34]: What would this do exactly? If the issue has "ver: 1.6.x" then it adds the "ver: 1.6.x" label?

File: .github/ISSUE_TEMPLATE/1_bug_report.yaml [hunk 3]
```diff
@@ -29,13 +29,16 @@ body:
...
       multiple: true
       options:
-        - 1.6.x
-        - 1.7.x
-        - 1.8.x
-        - 1.9.x
-        - 2.0+
+        - "ver: 1.6.x"
+        - "ver: 1.7.x"
+        - "ver: 1.8.x"
```
Comment (carmocca) [inline .github/ISSUE_TEMPLATE/1_bug_report.yaml:38]: Is having 1.6, 1.7 and 1.8 useful at all? If we only support 1.9.x and 2.x moving forward, those should be the only options.

Then there could be a "Does this reproduce on master" checkbox.

File: .github/labeler.yml (hunk 1 above)
Comment (Borda) [reply .github/labeler.yml:35]: That is fine, but I was thinking about how to automate it...

File: .github/workflows/issue-labeler.yml (hunk 2 above)
Comment (Borda) [reply .github/workflows/issue-labeler.yml:34]: yes, that is my expectation

File: .github/ISSUE_TEMPLATE/1_bug_report.yaml (hunk 3 above)
Comment (Borda) [reply .github/ISSUE_TEMPLATE/1_bug_report.yaml:38]: we can trace the root of the problem, and eventually after closing older issues...
having this information is not slowing us anyway...

File: .github/labeler.yml [hunk 4]
```diff
@@ -31,3 +31,6 @@ ci:
...
+  - 'docs/**'
```
Comment (Borda) [inline .github/labeler.yml:36]:
```suggestion
```

File: .github/labeler.yml (hunk 1 above)
Comment (Borda) [reply .github/labeler.yml]: ok, dropped for now :chipmunk:

**Review by lantiga — APPROVED** [review]

**Review by carmocca — APPROVED** [review]

**Review by carmocca — COMMENTED** [review]
Something's not right. Bug reports cannot be created

**Comment by awaelchli** [comment]
@carmocca it seems to work for me: #17496

**Comment by carmocca** [comment]
Jirka pushed a fix before you tried: 8684783a6eaa8f823fad52fd9d37b65119b2c055

**Pull request merged**